app/packages/looker/src/elements/common/tags.ts

```diff
@@ -426,46 +412,38 @@ const getFieldAndValue = (
   sample: Sample,
   schema: Schema,
   path: string
-): [Field | null, unknown, boolean] => {
+): [Field | null, unknown] => {
   let value: unknown = sample;
   let field: Field = null;
-  let list = false;
 
   for (const key of path.split(".")) {
     if (!schema?.[key]) {
-      return [null, null, false];
+      return [null, null];
     }
 
     field = schema[key];
 
     if (field && field.embeddedDocType === "fiftyone.core.frames.FrameSample") {
-      return [null, null, false];
+      return [null, null];
     }
 
     if (![undefined, null].includes(value) && field) {
       value = unwind(field.dbField, value);
-      list = list || field.ftype === LIST_FIELD;
     }
 
-    schema = field ? field.fields : null;
-  }
+    if (
+      field &&
+      field.embeddedDocType === withPath(LABELS_PATH, CLASSIFICATIONS)
+    ) {
+      value = value["classifications"] || [];
```

Comment:
This is breaking - you could optionally access it
```
value = value?.["classifications"] || [];
field = field.fields?.["classifications"];
```

<img width="1615" alt="Screen Shot 2023-05-30 at 1 28 21 PM" src="https://github.com/voxel51/fiftyone/assets/109545780/882ecbe1-3aec-4c52-8277-8dad61c467c6">